perllib/FixMyStreet/App.pm

```diff
@@ -81,6 +81,18 @@ __PACKAGE__->config(
                 user_model => 'DB::User',
             },
         },
+        access_token => {
+            use_session => 0,
+            credential => {
+                class => 'AccessToken',
+                token_field => 'extra',
+                token_lookup => { like => "%access_token,I20:TOKEN,%" },
+            },
```

Comment:
Might be worth a comment here that this means the token has to be 20 characters long, as much to document it somewhere.

Reply:
Will do, but random_token generates 18 char long token, which is fine, updating the count now.